runtime/android/core_internal/src/org/xwalk/core/internal/extension/api/contacts/ContactConstants.java

```diff
@@ -38,7 +38,7 @@
     // e.g. When to find "givenName=John", we actually need to find "John" in "givenNames".
     public static final Map<String, String> findFieldMap = createStringMap(new String[]{
         "familyName", "familyNames",
-        "givenName", "givenNames",
+        "givenNames", "givenNames",
```

Comment:
Sorry for reply you late.According to idl-def-ContactName in w3c and the crosswalk-test-suite,there is no use for "givenName".So we should change the function name mapping.